src/app/api/_api-services/onchain_db_service/subsquid_service/index.ts

```
@@ -1057,4 +1057,110 @@ export class SubsquidService extends SubsquidUtils {
 			totalCount: subsquidData.proposalsConnection.totalCount || 0
 		};
 	}
+
+	static async GetPostAnalytics({ network, proposalType, index }: { network: ENetwork; proposalType: EProposalType; index: number }): Promise<IPostAnalytics> {
+		const gqlClient = this.subsquidGqlClient(network);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)